cmake/modules/OmrCompilerSupport.cmake

```diff
@@ -298,6 +298,7 @@ function(create_omr_compiler_library)
 	target_link_libraries(${COMPILER_NAME}
 		PUBLIC
 			omr_base
+			${OMR_PORT_LIB}
```

Comment:
Curious where is this set exactly? Also @dnakamura any concerns?

Reply:
@fjeremic, it is set here:
https://github.com/eclipse/omr/blob/4c449812b348cb8fa1010a7ea8473c5bf0aa9dfb/cmake/config.cmake#L111